xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOMX.cpp

```diff
@@ -27,6 +27,7 @@
 #include "utils/GLUtils.h"
 #include "settings/MediaSettings.h"
 #include "windowing/WindowingFactory.h"
+#include "DVDCodecs/Video/DVDVideoCodecIMX.h"
```

Comment:
anything named "IMX" in this file is most likely wrong: for reference check: https://github.com/xbmc/xbmc/blob/Jarvis/xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp